lib/vast/Frontend/Consumer.cpp

```diff
@@ -188,16 +210,98 @@ namespace vast::cc {
             llvm::DebugFlag = true;
         }
 
+#ifdef VAST_ENABLE_SARIF
+        gap::sarif::run sarif_run{
```

Comment:
Make these to separate functions, as it makes `process_mlir_module` quite unreadable at the moment.